src/clients/c++/image_client.cc

```diff
@@ -468,7 +526,8 @@ void
 FileToInputData(
```

Comment:
This function should still just return the input_data. Make another function to get it into shared memory.